bitnami/thanos/templates/bucketweb/servicemonitor.yaml

```diff
@@ -8,6 +8,9 @@ metadata:
   {{- end }}
   labels: {{- include "thanos.labels" . | nindent 4 }}
     app.kubernetes.io/component: bucketweb
+{{- if .Values.metrics.serviceMonitor.labels }}
+{{ toYaml .Values.metrics.serviceMonitor.labels | indent 4}}
```

Comment:
Could you use `{{-` and `nindent` instead? With that you could indent this block to the same level as the rest of the labels, making the code more readable

Reply:
Yeah makes sense. Will change it